tickerv2/ticker.go

```diff
@@ -0,0 +1,93 @@
+package tickerv2
+
+import (
+	"fmt"
+	"github.com/fahimimam/busbdChckr/businfo"
+	avbusinfo "github.com/fahimimam/busbdChckr/businfo/availableBusInformation"
+	"github.com/fahimimam/busbdChckr/businfo/models"
+	"log"
+	"strings"
+	"time"
+)
+
+const StructureTypeBus = "BUS"
+
+func RunTickerV2() {
+	source := "Dhaka"
+	source = strings.ToLower(source)
+	destination := "khulna"
+	destination = strings.ToLower(destination)
+	data := avbusinfo.RequestPld{
+		Date:          "2024-04-04",
+		Identifier:    fmt.Sprintf("%s-to-%s", source, destination),
+		StructureType: StructureTypeBus,
+	}
+	// Initial fetch
+	prevData, err := fetchBusInfo(data)
+	if err != nil {
+		log.Println("Error fetching initial data:", err)
+	}
+	log.Println("Fetched Data from server...")
+	// Start a goroutine to periodically check for updates
+	ticker := time.NewTicker(1 * time.Hour) // Adjust interval as needed
+	defer ticker.Stop()
+	log.Println("Ticker Running...")
```

Comment:
Can we use `slog` for logging? It's like `logrus` but from Go standard library...